.github/pull_request_template.md

```diff
@@ -4,6 +4,15 @@
   - Note:
     - Consider running `automation/all_tests.sh` to execute these tests locally.
     - For changes that need extra cross-platform testing, consider adding `[ci full]` to the PR title.
+  - Run smoke tests for iOS, android-components, and fenix to ensure the release will integrate cleanly with key downstream consumers
+    - Run the `./automation/smoke-test-firefox-ios.py` script to test integration with Firefox for iOS.
```

Comment:
Oh man . . . yeah CI doesn't run any of these scripts. I'm going to add a note here and an agenda item for our next kitchen sync. Code committers should have some means to verify code for consumers but this may not be it.